pydevices/HtsDevices/acq2106_WRPG.py

```diff
@@ -0,0 +1,189 @@
+#
+# Copyright (c) 2017, Massachusetts Institute of Technology All rights reserved.
```

Comment:
Change this to Copyright 2020

Reply:
Done. Thanks. I also changed it in Sean's code.